app.rb

```diff
@@ -76,7 +76,7 @@ def perform(country_slug, legislature_slug, source = nil)
     end
     # Only use last 64k of output
     output = output[-64_000..-1] || output
-    title = "#{country.name} (#{legislature.name}): refresh data"
+    title = "#{country.name} (#{legislature.name}#{source ? ":#{source}" : ''}): refresh data"
     body = "Automated data refresh for #{country.name} - #{legislature.name}" \
```

Comment:
The text here is slightly odd too. This will give us, for example "South Korea (National Assembly:wikidata): refresh data"

After some discussion on slack, we've decided on:

"South Korea (National Assembly): refresh wikidata"

and the body:

"Automated refresh of Wikidata for South Korea - National Assembly"

with, in both cases, a fallback of "all data" if there's no source.